packages/mobx-state-tree/src/core/type/type.ts

```diff
@@ -520,6 +518,7 @@ export abstract class SimpleType<C, S, T> extends BaseType<C, S, T, ScalarNode<C
         return null
     }
 }
+BaseType.prototype.create = action(BaseType.prototype.create)
```

Comment:
Maybe this row should be placed directly below the `BaseType` class?

Reply:
Is it not? These classes are so long I had a hard time locating where they end :)

Reply:
This is below `SimpleType`, I believe! The `BaseType` class definition ends at line ~370.

Reply:
Good call, should be fixed, thanks!